src/components/Admin/DropDown.jsx

```diff
@@ -12,30 +12,36 @@ const Toggle = ({ children, onClick }) => {
     </button>
   );
 };
-const Menu = ({ children, className }) => {
+const Menu = ({ children, className, setOptions, options }) => {
   const [value, setValue] = useState("");
   return (
     <div className={className}>
       <input
         autoFocus
         className="mx-3 my-2 w-fit ring-0 outline-none"
         placeholder="Type to filter..."
-        onChange={(e) => setValue(e.target.value)}
+        onChange={(e) => {
+          setValue(e.target.value);
+          setOptions(
+            options.filter(
+              (option) => !value || option.toLowerCase().startsWith(value)
```

Comment:
do we need the toLowerCase(), we should be able to prevent this situation since its done by us only